core/explorer.go

```diff
@@ -84,6 +152,12 @@ func (c *Core) InitRubixExplorer() error {
 		return err
 	}
 
+	err = c.s.Init(ExplorerUserDetailsTable, &ExplorerUser{}, true)
+	if err != nil {
+		c.log.Error("Failed to initialise storage ExplorerUserDetails ", "err", err)
+		return err
+	}
+
 	url := "deamon-explorer.azurewebsites.net"
```

Comment:
shouldnt these URL be coming from explorer db?

Reply:
It is coming from db when making requests. But while initialising the ExplorerURLTable, its adding the mainnet or testnet explorer link. This was there since earlier. Didn't make any changes here. Should I remove these links?